app/views/build_estimates/_money_input.html.slim

```diff
@@ -0,0 +1,16 @@
+- if hint_text.present?
+  = form.govuk_text_field field,
+          width:,
+          hint: { text: hint_text },
+          label: { text: label_text, size: "m" },
+          prefix_text: BuildEstimatesHelper::POUND,
+          pattern: "[0-9]*",
+          inputmode: "numeric"
```

Comment:
This may be _way_ beyond the scope of this ticket, but since we're changing this field: I think it's weird that if I enter `500` here then click back, what I see is `500.0`. (Because `#to_s` on an integer value stored as a BigDecimal always appends `.0` )

I think a monetary field should either have zero decimal places or two decimal places, but not one. I can't see an _easy_ fix for this, the best I can come up with is to insert a `value: decimal_as_money_string(form, field)` here, and then adding a helper method along the lines of:

```ruby
  def decimal_as_money_string(form, field)
    current_value = form.object.attributes[field.to_s]
    return if current_value.blank?
    return current_value.to_i if current_value.round == current_value

    sprintf("%.2f", form.object.attributes[field.to_s])
  end
```

Massive overkill?

Reply:
I've decided that the best thing to do is to stop digging here. I've created a new ticket EL-377 to look at this whole area